src/Microsoft.ML.Api/SchemaDefinition.cs

```diff
@@ -287,9 +287,9 @@ public void RemoveMetadata(string kind)
         /// </summary>
         public Column this[string columnName]
         {
-#pragma warning disable TLC_NoThis // Do not use 'this' keyword for member access
+#pragma warning disable MSML_NoThis // Do not use 'this' keyword for member access
```

Comment:
:memo: This analysis is available through a couple of alternatives, which may eliminate sources of bugs.

Reply:
Replaced with [SX1101](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SX1101.md).